ehr/resources/pipeline/kinship/populateInbreeding.r

```diff
@@ -30,6 +31,9 @@ is.na(allPed$Gender)<-which(allPed$Gender=="")
 df <- data.frame(id=as.character(allPed$Id), 'id parent1'=allPed$Dam, 'id parent2'=allPed$Sire, stringsAsFactors=FALSE);
 colnames(df)<-c("id", "id parent1", "id parent2")
 
+originalIds <-as.data.frame(df[,1])
```

Comment:
originalIds <-df[,1,drop=FALSE]

does does this. drop=FALSE preserves the single-column vector as a data.frame. you dont need to set colnames() then either.

Reply:
Nice, works great. Thanks.